.scalafmt.conf

```diff
@@ -24,7 +24,13 @@ runner.dialect = scala3
 project {
   excludePaths = [
     "glob:**/example/scalalib/linting/1-scalafmt/src/Foo.scala",
-    "glob:**/mill/out/**"
+    "glob:**/scalalib/test/resources/checkstyle/**",
+    "glob:**/mill/out/**",
+    "glob:**/scalalib/test/resources/giter8/hello.g8/src/main/g8/build.mill",
+    "glob:**/scalalib/test/resources/javalib/palantirformat/**",
+    "glob:**/integration/failure/parse-error/**",
+    "glob:**/testkit/test/resources/example-test-example-project/build.mill",
+    "glob:**/scalalib/test/resources/scalafmt/core/src/**"
```

Comment:
I think we should leave comments why these are excludes, to guide the maintainer. I think [`scalafmt.conf` is HOCON](https://scalameta.org/scalafmt/docs/configuration.html), not JSON.